pom.xml

```diff
@@ -142,10 +142,7 @@ SOFTWARE.
       <plugin>
         <artifactId>maven-compiler-plugin</artifactId>
         <version>3.8.1</version>
-        <configuration combine.self="override">
-          <source>11</source>
-          <target>11</target>
-        </configuration>
```

Comment:
@Graur, Here we use <target> and <source>, since there are methods (https://github.com/objectionary/dejump/blob/master/src/main/java/org/eolang/dejump/RemoveGoto.java#L129-L133), 
which requires language level 11

Reply:
`Files.readString()` :

![image](https://user-images.githubusercontent.com/95661085/198041399-20cf37cd-3bad-4f03-95d2-01d315a11508.png)

Reply:
@Graur, does it work correctly in your IDE without specifying the language level in `pom.xml` ?

Reply:
@MikhailLipanin Yes it does. Moreover, github checks are passes too

Reply:
@MikhailLipanin But I can't find out why gh-pages check failed. Did it work in the previous builds?

Reply:
@Graur, Here, I don't know why ;( In my IDE it says this error:

![image](https://user-images.githubusercontent.com/95661085/198042834-9c9dae66-7ed3-4afc-9b2e-c95c613bcc01.png)

Reply:
@MikhailLipanin Change it here:
<img width="1009" alt="Screenshot 2022-10-26 at 16 53 32" src="https://user-images.githubusercontent.com/33455325/198044850-3995e60e-a572-46b0-8e39-31d8a88dc2bd.png">

Reply:
@Graur, Thanks! It helped